ajax/libs/popper.js/package.json

```
@@ -1,11 +1,11 @@
 {
   "name": "popper.js",
   "filename": "popper.min.js",
```

```suggestion
  "filename": "umd/popper.min.js",
```
Whilst we're here, let's update this to reflect what's in the package.